pkg/registry/manager.go

```diff
@@ -60,6 +60,12 @@ func Locate(a app.App, spec *app.RegistryRefSpec) (Registry, error) {
 		return githubFactory(a, spec)
 	case ProtocolFilesystem:
 		return NewFs(a, spec)
+	case ProtocolHelm:
```

Comment:
Is `registry add` support in scope for this PR?

Reply:
Nope. Don't want to do that until we have a complete solution.